generators/typescript/codegen/src/ast/Type.ts

```
@@ -29,4 +136,37 @@
             type: "string"
         });
     }
+
+    public static number(): Type {
+        return new this({
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)